src/update-depth-charts.R

```diff
@@ -82,29 +81,38 @@ build_dc <-
             game_type == "POST" &
               week == 3 ~ "CON",
             game_type == "POST" &
-              week == 4 ~ "SB",
+              ((
+                week == 4 & season %in% c(2002:2006, 2014)
+              ) |
+                (week == 5)) ~ "SB",
+            game_type == "POST" &
+              (week == 4 &
+                 !(season %in% c(2002:2006, 2014))) ~ "SBBYE",
             T ~ game_type
           ),
           week = dplyr::case_when(
-            game_type %in% c("WC", "DIV", "CON", "SB") ~ week + max(week[game_type == "REG"]),
-            T ~ week
-          ),
-
+            game_type == "SBBYE" ~ NA_integer_,
```

Comment:
It might be helpful to leave a comment about what’s going on here eg aligning to nflverse game week numbers